wpilibNewCommands/src/test/native/cpp/frc2/command/MecanumFollowerCommandTest.cpp

```diff
@@ -0,0 +1,119 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+#include <frc2/command/Subsystem.h>
+#include <frc2/command/MecanumFollowerCommand.h>
+#include <frc/controller/PIDController.h>
+#include <frc/controller/ProfiledPIDController.h>
+#include <frc/kinematics/MecanumDriveKinematics.h>
+#include <frc/kinematics/MecanumDriveOdometry.h>
+#include <frc/geometry/Translation2d.h>
+#include <frc/geometry/Rotation2d.h>
+#include <frc/geometry/Rotation2d.h>
+#include <frc2/Timer.h>
+#include <wpi/math>
+#include "frc/trajectory/TrajectoryGenerator.h"
+#include "gtest/gtest.h"
+#include <iostream>
+
+#define EXPECT_NEAR_UNITS(val1, val2, eps) \
+  EXPECT_LE(units::math::abs(val1 - val2), eps)
+
+class MecanumFollowerCommandTest : public ::testing::Test {
+protected:
+  frc2::Timer m_rotTimer;
+  frc::Rotation2d m_angle{0_rad};
+
+  units::meters_per_second_t  m_frontLeftSpeed = 0.0_mps;
+  units::meters_per_second_t  m_rearLeftSpeed = 0.0_mps;
+  units::meters_per_second_t  m_frontRightSpeed = 0.0_mps;
+  units::meters_per_second_t  m_rearRightSpeed = 0.0_mps;
+
+  frc::ProfiledPIDController m_rotController{1, 0, 0,
+      frc::TrapezoidProfile::Constraints{9_mps, 3_mps_sq}};
+
+  static constexpr units::meter_t kxTolerance{1 / 12.0};
+  static constexpr units::meter_t kyTolerance{1 / 12.0};
+  static constexpr units::radian_t kangularTolerance{1 / 12.0};
+
+
+  static constexpr units::meter_t m_trackLength{0.5};
+  static constexpr units::meter_t m_trackWidth{0.5};
```

Comment:
Aren't these technically constants? Meaning that they should start with `k` instead of `m_`.